front/app/utils/dateUtils.ts

```diff
@@ -189,31 +159,26 @@ export function toFullMonth(date: string, resolution: IResolution) {
     .format(resolution === 'month' ? 'MMMM YYYY' : 'MMMM DD, YYYY');
 }
 
-// This function is used to display event start/end times with weekday names
-export const getEventDateWithWeekdays = (
-  event: IEventData,
-  formatMessage: (
-    messageDescriptor: MessageDescriptor,
-    values?:
-      | {
-          [key: string]: string | number | boolean | Date;
-        }
-      | undefined
-  ) => string
-) => {
-  const startAtMoment = moment(event.attributes.start_at);
-  const endAtMoment = moment(event.attributes.end_at);
+// Function used to determine whether a dot should be shown after the day in short date formats
+// Currently only used for German. Other locales can be added if needed.
+export function showDotAfterDay(locale: Locale) {
+  return locale === 'de-DE';
+}
+
+// Function used to get the event dates in a localized string format
+export function getEventDateString(event: IEventData) {
```

Comment:
I chatted with Simon F about this and we decided to just simplify the event dates to use the moment library only (so no weekday names) so we're sure the solution works for every locale :)